allofplos/article_class.py

```diff
@@ -842,7 +875,23 @@ def proof(self):
         return proof
 
     @property
-    def remote_element_tree(self):
+    def remote_proof(self):
+        """
+        For a single article online, check whether it is an 'uncorrected proof' or a
+        'VOR update' to the uncorrected proof, or neither.
+        :return: proof status if it exists; otherwise, None
```

Comment:
isn't it otherwise an empty string `''`?

Reply:
fixed in https://github.com/PLOS/allofplos/pull/54/commits/14313bd6bf1fc586317242f3a48734ad30711fa7